src/inverseRobot.js

```diff
@@ -25,6 +25,18 @@
  */
 function inverseRobot(robot) {
   // write code here
+  let reversObj = {};
+
+  for (const key in robot) {
+    if (reversObj[robot[key]]) {
+      reversObj = null;
+
+      return reversObj;
+    }
```

Comment:
```suggestion
    if (reversObj[robot[key]]) {
      return null;
    }
```
Просто нічого не повертай. Для чого присвоювати в змінну null, а потім її повертати, якщо можна просто повернути null?
Це не зовсім те, що потрібно для цієї задачі

Reply:
я виправив код, будь-ласка, перегляньте